app/controllers/hyrax/admin/admin_sets_controller.rb

```diff
@@ -55,7 +55,8 @@ def files
 
     def update
       if @admin_set.update(admin_set_params)
-        redirect_to hyrax.edit_admin_admin_set_path(@admin_set), notice: I18n.t('updated_admin_set', scope: 'hyrax.admin.admin_sets.form.permission_update_notices', name: @admin_set.title.first)
+        referer = hyrax.edit_admin_admin_set_path(@admin_set) + (params['referer_anchor'] || '')
```

Comment:
I realize this is a simpler case than above, but I might like to see this handled in a new private method, `update_referer`. Also prefer to use a symbol for the hash key if possible.